### PR TITLE
Adding Constenla-2005

### PR DESCRIPTION
**Pull request by Tarotis**
# Pull request checklist

- [x]  add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
This PR aims at adding the 110-item concept list offered by Constenla Umaña (2005). No new concepts have been introduced. As this is my first PR to Concepticon I hope I got everything right. If not I will try to adapt any necessary change asap.
...

### CODE REVIEW

**Comment by Tarotis** [comment]
@LinguList I created the PR according to the guidelines in the blog, but for some reason the fest fails, claiming inconsistent data. I tried to pin it down by checking every step of the mapping process, but I can't seem to find it. What does this error message usually refer to?

**Comment by LinguList** [comment]
fixed it, the metadata should give type "string" for the field "NUMBER", as the number can also contain "ABC", etc., and the check is sensitive to this :)

**Comment by LinguList** [comment]
I'll moderate this PR and would ask @Schweikhard and @chrzyki to have a quick look during next week, if possible.

File: concepticondata/conceptlists/Constenla-2005-110.tsv
Comment (Schweikhard) [inline concepticondata/conceptlists/Constenla-2005-110.tsv:97]: Considering the English gloss, this is probably mapped correctly, but all other instances of caer in the database seem to be mapped to FALL instead.

File: concepticondata/conceptlists/Constenla-2005-110.tsv
Comment (Schweikhard) [inline concepticondata/conceptlists/Constenla-2005-110.tsv:56]: I think saber is more specific than know and only refers to KNOW (SOMETHING).

File: concepticondata/conceptlists/Constenla-2005-110.tsv
Comment (Schweikhard) [inline concepticondata/conceptlists/Constenla-2005-110.tsv:14]: CLEAN refers to the property. Depending on the exact meaning, this should be mapped to CLEAN (SOMETHING), WASH, or LAUNDER (WASH CLOTHES). We have thus far mapped lavar to WASH.

File: concepticondata/references/references.bib
Comment (Schweikhard) [inline concepticondata/references/references.bib:3867]: I would suspect this should start with ¿ rather than ?`.

File: concepticondata/conceptlists.tsv
Comment (Schweikhard) [inline concepticondata/conceptlists.tsv:377]: "a reduced form _of_" sounds more correct to me.

**Review by Schweikhard — COMMENTED** [review]
Thank you for contributing this list! The mapping and everything looks good, just some small corrections:

**Review by chrzyki — APPROVED** [review]
Likewise, many thanks for adding this. I don't have anything to add except for what @Schweikhard already mentioned. Is there any reason for the order of the glosses/concepts in the list? Why not order it by ascending NUMBER/ID?

File: concepticondata/references/references.bib
Comment (Tarotis) [reply concepticondata/references/references.bib:3867]: As far as I know, both usually work, but I guess its more transparent if `¿` is used.

**Comment by Tarotis** [comment]
Just to be sure on the workflow because this is the very first PR I am working on in git: I adapt and commit the suggested changes, and afterwards Mattis merges the PR?

As for the ordering, there is no special reason for this. Would it make more sense to order according to Concepticon_ID or is this not uniformly formatted across the other lists and Number (the original authors order in his paper) works just as well?

**Comment by chrzyki** [comment]
> 
> 
> Thanks for the kind replies and suggestions. Just to be sure on the workflow because this is the very first PR I am working on in git: I adapt and commit the suggested changes, and afterwards Mattis merges the PR?
> 
> As for the ordering, there is no special reason for this. Would it make more sense to order according to Concepticon_ID or is this not uniformly formatted across the other lists and Number (the original authors order in his paper) works just as well?

Yes, you can incorporate the changes and then push the updated branch. Afterwards, they'll appear here in this PR and we can approve/merge.

For the ordering: Yes, generally speaking, we try to keep the order as in the original list and the resulting TSV should be in ascending order for the NUMBER/ID column. Makes for an easier reading.

**Comment by Tarotis** [comment]
Thanks for the fast reply! I now updated the files with the corresponding changes.
In order to comply with the original author's ordering (Spanish alphabet), a rewrite of the number-column was necessary. Actually, you are not only totally correct in that it improves readability, but it also improves comparability if somebody wants to cross-check the lexibank-database with the original paper, so thanks a lot for that suggestion.

**Comment by LinguList** [comment]
I'll merge later, once I have reviewed the whole discussion and see no need for further changes. BTW: is the original text with glosses in English and Spanish or only Spanish?

**Comment by Tarotis** [comment]
The original glosses are only given in Spanish

**Comment by LinguList** [comment]
So the column ENGLISH should be renamed to column GLOSS, as you made the 
translation, right? In that case, I'd even suggest to go lowercase, as 
you can give a translation that also reflects part of speech, etc., and 
is a bit more verbose. E.g., you can indicate if something is a verb, 
right? Otherwise, bailar is ambiguos as gloss in "dance".

**Comment by LinguList** [comment]
Ah, just to be sure, could you send me the PDF via email, so I can have 
a look?

**Comment by Tarotis** [comment]
I just sent you a download link per E-Mail (file was too big). Should I adapt the changes to ENGLISH/GLOSS already or wait for your confirmation? The translation is mine, so all of the suggestions can be adapted easily.

**Comment by LinguList** [comment]
Thanks, I received the PDF now, this is important, as you will have to modify the Spanish glosses as well. You give them in upper case fully, but the source only capitalizes the first letter, so you should adhere to this. If it is not too much work, you could even add the page number in an additional column, which is helpful for those who want to look at the original resource. The English glosses are free, but I recommend to disambiguate things like "dance", so it is clear which POS we are dealing with.

**Comment by Tarotis** [comment]
Thanks for the suggestions, I changed the respective columns and included a new column `PAGE_IN_SOURCE` to the list and the metadata.

**Review by LinguList — APPROVED** [review]

**Comment by LinguList** [comment]
I just modified the glosses, will then merge.

**Pull request merged**
